conf/base.config

```diff
@@ -47,5 +46,7 @@ process {
   withName:get_software_versions {
     cache = false
   }
-  
+  withLabel:py3{
+  	 container = 'barryd237/py3:dev'
+  	}
```

Comment:
What is going on here? This looks suspicious.. 😰

Reply:
The container runs on Python 2.7, it was one of FastQC or MultiQC (and a quantification tool DCC) that would not run on python 2.7 hence the python 3 container.